NumLib/DOF/GlobalMatrixProviders.h

```diff
@@ -14,17 +14,14 @@
 
 namespace MathLib
 {
-
-template<typename Vector>
 struct GlobalVectorProvider
 {
-    static VectorProvider<Vector>& provider;
+    static VectorProvider<GlobalVector>& provider;
```

Comment:
Should be possible since the Eigen::MatrixXd cases are removed from TestODEInt.cpp